### PR TITLE
Frontend base

### PR DESCRIPTION
**Pull request by karcagtamas**

### CODE REVIEW

File: frontend/.prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "trailingComma" : "none"
```
Comment (karcagtamas) [inline frontend/.prettierrc.json:2]: The single quote is working correctly @Cerberuuus?

**Comment by karcagtamas** [comment]
@Cerberuuus, @bencebaranyai I approve it, but I am the PR creator.

File: frontend/.prettierrc.json (hunk 1 above)
Comment (Cerberuuus) [reply frontend/.prettierrc.json:2]: Yes, as it is in the _tslint.json_.
```json
 "quotemark": [
      true,
      "single"
    ],
```

However, the above configuration may not be needed, if you think that trailing commas are not confusing and shouldn't be ignored.

Comment (karcagtamas) [reply frontend/.prettierrc.json:2]: Yes, I thought to the default formatter settings. But if you do not found a quote problem, then it is ok.

**Review by bencebaranyai — APPROVED** [review]

**Pull request merged**